README.md

```diff
@@ -142,6 +142,14 @@ Please post your problem in [our free support forum](http://community.rtcamp.com
 
 ## Changelog ##
 
+### 2.2.0 ###
+* Add filter `rt_nginx_helper_fastcgi_purge_suffix` to change purge suffix for FastCGI cache. [#141](https://github.com/rtCamp/nginx-helper/pull/141) - by [stayallive](https://github.com/stayallive)
+* Add filter `rt_nginx_helper_fastcgi_purge_url_base` to change purge URL base for FastCGI cache. [#141](https://github.com/rtCamp/nginx-helper/pull/141) - by [stayallive](https://github.com/stayallive)
+* Update our code to be in line with WordPress Coding standards in various places. [#209](https://github.com/rtCamp/nginx-helper/pull/209) - by [abhijitrakas](https://github.com/abhijitrakas)
```

Comment:
Also mention https://github.com/rtCamp/nginx-helper/pull/225/ PR so we can have all the PRs in change log for PHPCS error fixing.

Reply:
Updated in https://github.com/rtCamp/nginx-helper/pull/226/commits/fa989a9538a8ae50c1bd97bf6cc1cc61251b50a2